tutorials/01-gaussian-mixture-model/index.qmd

```diff
@@ -109,14 +106,20 @@ We run a MCMC simulation to obtain an approximation of the posterior distributio
 We use a `Gibbs` sampler that combines a [particle Gibbs](https://www.stats.ox.ac.uk/%7Edoucet/andrieu_doucet_holenstein_PMCMC.pdf) sampler for the discrete parameters (assignments $k$) and a Hamiltonion Monte Carlo sampler for the continuous parameters ($\mu$ and $w$).
 We generate multiple chains in parallel using multi-threading.
 
-```julia
+```{julia}
+#| output: false
+setprogress!(false)
+```
+
+```{julia}
 sampler = Gibbs(PG(100, :k), HMC(0.05, 10, :μ, :w))
```

Comment:
```suggestion
#| output: false
sampler = Gibbs(PG(100, :k), HMC(0.05, 10, :μ, :w))
```

we can suppress this warning and add a callout block underneath the code cell to explain. I don't think 
`#| warning: false` really works in julia quarto right now, so let's just use `output: false`

Reply:
In other notebooks, code blocks are generating other output too along with that warning so is it fine to use `output: false` with all code blocks with this warning

Reply:
Hmm I think in those cases, we could split those code blocks up? We can do something like:

```{julia}
#| output: false
chains = sample(stuff ...)
```

```{julia}
#| echo: false
chains # Display chains
```
This should look the exact same as if we only suppressed the warning.

I don't want to do anything too confusing, but I think this is not too bad ... if there's a standard, single-use way to display a variable in Quarto, we could use that instead. But I don't know if it exists.

Reply:
> single-use way to display a variable in Quarto

do you mean these variables? - https://quarto.org/docs/authoring/variables.html

Reply:
ohk, let me see! I will update other tutorials with this two-block structure